app/pdl.yaml

```diff
@@ -617,6 +617,386 @@ paths:
           description: Unauthorized
         500:
           description: Internal error
+  /pdl/titles/title-i/subtitles/subtitle-a/state-distribution:
```

Comment:
Please update the version at the top of the YAML file.

Reply:
updated.

Reply:
Thanks.